js/ui/projectManager.js

```diff
@@ -129,9 +129,18 @@ define([
     this.expanded = {};
     this.project = null;
     this.projectFile = null;
+
     if (element) {
       this.setElement(element);
+
+      // use host document object to bind resize events
+      if (document && !document.boundSidebarRersizeEvents) {
```

Comment:
Yep, cool, added it only because was not sure if it does ever get instantiated more then once.
Will remove the flag (sorry about the typo as well).